.eslintrc.js

```diff
@@ -18,6 +18,8 @@ module.exports = {
     browser: true
   },
   rules: {
+    'ember/avoid-leaking-state-in-ember-objects': 'off',
```

Comment:
@nightsh I'm not sure about removing `ember/avoid-leaking-state-in-ember-objects`.

According to [the docs](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-ember-objects.md):

> Don't use arrays and objects as default properties. More info here: https://dockyard.com/blog/2015/09/18/ember-best-practices-avoid-leaking-state-into-factories

That sounds sensible. Wouldn't it be wiser to just fix/avoid those instead of removing these?

Reply:
Generally, yes. But in our case, fixing this would very likely introduce new and complicated bugs, as we have lots of arrays and objects defined as default properties. This is something Ember itself supported well until one day.

However, I will create an issue and revisit this, I'm also pissed by having mixed class / instance properties in the same place. In the meantime, this is not a critical issue (i.e. not making Elvis buggy), while the rule denying a widely used pattern is making the build invariably fail, thus blocking also any automated testing/CI/CD efforts.